fern/docs/pages/getting-started/quick-start.md

```
@@ -0,0 +1,478 @@
+The simplest approach for getting started. Agents exchange metadata directly over a TCP connection. One agent listens for incoming metadata, while the other fetches and sends.
+
+<CodeBlocks>
+```python title="Python"
+# On the initiator side:
+# Fetch the target's metadata (target is listening on ip:port)
+agent.fetch_remote_metadata("target", target_ip, target_port)
+
+# Send this agent's metadata to the target
+agent.send_local_metadata(target_ip, target_port)
+
+# Wait for metadata to be available
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)